Makefile

```diff
@@ -22,7 +22,7 @@ DIRS := \
   contrib/broker/server \
   controller \
   pkg \
-  pkg/api
+  pkg/apis/servicecatalog
```

Comment:
Hmm ... I don't buy this argument. In face of other instances where k8s leaders told us about other mistakes we shouldn't repeat we didn't heed their advice, but in this (equally arbitrary) case we do? Please explain.